_licenses/gfdl-1.3.txt

```diff
@@ -0,0 +1,476 @@
+---
+title: GNU Free Documentation License 1.3
+nickname: GNU FDL 1.3
+spdx-id: GFDL-1.3
+
+description: A free but copyleft license for software documentation that allows almost any use. The Free Software Foundation does not reccommend using the GPL on documentation. Contributors provide an express grant of patent rights.
+how: Create a text file (typically named COPYING, as per GNU conventions) in the root of your source code and copy the text of the license into the file.
+
+note: The Free Software Foundation recommends taking the additional step of adding a boilerplate notice to the top of each file. The boilerplate can be found at the end of the license.
+
+permissions:
+  - commercial-use
+  - modifications
+  - distribution
+  - patent-use
+  - private-use
+conditions:
+  - include-copyright
+  - disclose-source
+  - same-license
+  - document-changes
+limitations:
+  - liability
+  - warranty
+---
```

Comment:
Need to add 3 example uses, see other licenses for the format.

Reply:
During a workaround, I am marking this pull request as draft.

Reply:
Code search the following: [GNU FREE DOCUMENTATION LICENSE](https://github.com/search?q=GNU+FREE+DOCUMENTATION+LICENSE&type=code)

Reply:
Here's a narrower search more likely to find candidate examples https://github.com/search?q=filename%3ALICENSE+%22GNU+Free+Documentation+License+Version+1.3%2C+3+November+2008%22&type=code or https://github.com/search?q=filename%3ACOPYING+%22GNU+Free+Documentation+License+Version+1.3%2C+3+November+2008%22&type=code

Reply:
> Here's a narrower search more likely to find candidate examples https://github.com/search?q=filename%3ALICENSE+%22GNU+Free+Documentation+License+Version+1.3%2C+3+November+2008%22&type=code or https://github.com/search?q=filename%3ACOPYING+%22GNU+Free+Documentation+License+Version+1.3%2C+3+November+2008%22&type=code

[Here's](https://github.com/search?l=Text&q=filename%3ALICENSE+%22GNU+Free+Documentation+License+Version+1.3%2C+3+November+2008%22&type=Code) an extra narrower search with only text files.

Reply:
Added `using:` above under which examples should be added, whenever someone finds some.